index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+
+    <!-- font awesome -->
+    <script src="https://kit.fontawesome.com/833e2db6e5.js" crossorigin="anonymous"></script>
+
+    <!-- Google Font Montserrat Prompt -->

```

Comment:
It’s better to delete commented code. If you need to review a previous version of your code, use git history